app/Models/Asset.php

```diff
@@ -99,16 +99,17 @@ public function declinedCheckout(User $declinedBy, $signature)
         'expected_checkin' => 'date|nullable',
         'location_id'     => 'exists:locations,id|nullable',
         'rtd_location_id' => 'exists:locations,id|nullable',
-        'asset_tag'      => 'required|min:1|max:255|unique_undeleted:assets,asset_tag|not_array',
+        'asset_tag'       => 'required|min:1|max:255|unique_undeleted:assets,asset_tag|not_array',
         'purchase_date'   => 'date|date_format:Y-m-d|nullable',
-        'serial'          => 'unique_serial|nullable',
+        'serial'          => 'unique_undeleted:assets,serial|nullable',
```

Comment:
Furthermore, I *also* suspect that the 'un-stomped-on' validations may not be correct when running in FormRequest context. But it doesn't actually matter, since when the `$blah->save()` method fires, the model-level validation will 'catch' the validation failure near the end.